SecurityPkg/Include/Library/Tpm2CommandLib.h

```diff
@@ -973,6 +973,26 @@ Tpm2PolicyCommandCode (
   IN      TPM_CC          Code
   );
 
+// MS_CHANGE [BEGIN] - Add support for Tpm2PolicyLocality assertions.

```

Comment:
Can you adjust all the commit messages to be edk2 style (and drop old work items?)